Tests/UnitTests/Mocks/MockOfferingsManager.swift

```diff
@@ -63,4 +63,18 @@ class MockOfferingsManager: OfferingsManager {
         completion?(stubbedUpdateOfferingsCompletionResult!)
     }
 
+    var invokedInvalidateAndReFetchCachedOfferingsIfAppropiate = false
+    var invokedInvalidateAndReFetchCachedOfferingsIfAppropiateCount = 0
+
+    // swiftlint:disable identifier_name
```

Comment:
This is just my technique, if you're disabling a rule next to the code that breaks it I'd either re-enable it afterwards or do `disable:next`. Otherwise I would move this to the top of the file.

Reply:
Makes sense.  I used this technique because this code is at the end of this file and as far as I know this disable only affects the code below.  However, I agree with you that we need to re-enable this rule after code because we may add more code in the future that would be affected by this disable.  Let me change it 👍🏼